src/app/navbar/navbar.component.ts

```diff
@@ -47,4 +59,40 @@ export class NavbarComponent implements OnInit {
       this.isCollapsed = true;
     }
 
-}
+    getTemperature(): void { 
+
+      const API_KEY = "bc6a73dfabbd4e6c9006a835d00589f2";
+      const zipcode = "90024";
+      const baseWeatherUrl = "http://api.openweathermap.org/data/2.5/weather";
+      const defaultTemperatureUnits = "imperial";
+
+      var weatherQuery = `${baseWeatherUrl}?units=${defaultTemperatureUnits}&zip=${zipcode},us&APPID=${API_KEY}`;
```

Comment:
use `let` instead of `var`. `var` is function scoped and `let` is block scoped. Basically, `var` ambiguously acts as a global variable. This isn't consistent in the code yet, so that's a future thing to improve later